app/lib/three_scale/spam_protection/protector.rb

```diff
@@ -60,7 +60,7 @@ def captcha_required?
       end
 
       def captcha_needed?
-        captcha_required? || (!http_method.get? && spam?)
+        captcha_required? || spam?
```

Comment:
This is not correct, because it tries to render the captcha even if it is not configured, but it is like this in the code for now because I am only testing 😄